content/docs/user-guide/visualization/cinematics/track-view/nodes-material.md

```diff
@@ -19,7 +19,6 @@ A recommended workflow is to select the material inside the **Material Editor**
 1. Navigate to the material that you want to animate.
 
 1. Right-click the selected material and choose **Copy Path to Clipboard**. If the material is in a material group, select the group and copy the group name to the clipboard.
-
 ![Copy the material name path in the Material Editor.](/images/user-guide/cinematics/cinematics-track-view-nodes-material-4.png)
```

Comment:
```suggestion

    ![Copy the material name path in the Material Editor.](/images/user-guide/cinematics/cinematics-track-view-nodes-material-4.png)
```
Images and the opening tag of shortcodes like {{< note >}} need to be indented 4 spaces to be properly aligned and spaced in lists.